src/main/kotlin/no/nav/syfo/sykmelding/kafka/model/ReceivedSykmeldingMapper.kt

```diff
@@ -55,19 +53,6 @@ fun ReceivedSykmelding.toEnkelSykmelding(): EnkelSykmelding {
     )
 }
 
-private fun MedisinskVurdering.getHarRedusertArbeidsgiverperiode(): Boolean {
```

Comment:
Lå ikke denne sjekken flere steder..? Vi sender jo ut dette som respons i apiet også?

Reply:
har oppdatert nå, tror det bare var 1 plass vi gjordet det? så må vi også gjøre det nå for mottatte sykmeldinger når vi mapper om fra ReceivedSykmelding

Reply:
Ja, det var vel de to stedene du har endret nå bare? :) 

Burde vi hatt en test på dette forresten..?